philippa-choong/week2/tue/index.html

```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+     <link rel="stylesheet" href="css/style.css"> 
+    <title>Tim Berners-Lee Home Page</title>
+</head>
+
+<body>
+    <div class="container"> 
+        <!-- <nav class = "navbar">
+            <image src ="images/earth logo.png" class=" logo"></image> -->
+            <nav>
+                <ul>
+                    <li> <a href="about.html">ABOUT</a></li>
+                    <li> <a href="">ACHIEVEMENTS</a></li>
+                    <li> <a href="current.html">CURRENT WORKS</a></li>
+                </ul>
+            </nav>
+            <img    src="images/menu.png" class="menu-icon">
+        </nav>
+        <div class=" row">
```

Comment:
Coding Convention:
don't leave spaces between the values. 
so it should look like `<div class="row">`